src/components/ArticleView/AllArticles.jsx

```diff
@@ -0,0 +1,36 @@
+import ArticlePreviewCard from "./ArticlePreviewCard";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+
+const AllArticles = () => {
+  const [articlesAll, setArticlesAll] = useState([]);
+  React.useEffect(() => {
```

Comment:
as you've have desctructured import for useEffect you don't need the React. prefix